src/v2/Apps/Consign/Components/__tests__/SoldRecently.jest.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import { MockBoot } from "v2/DevTools"
+import { Breakpoint } from "@artsy/palette"
+import { useTracking } from "react-tracking"
+import { tests } from "../SoldRecently"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+jest.mock("v2/Components/Artwork/FillwidthItem", () => () => {
+  const FillwidthItem = () => <div />
+  return <FillwidthItem />
+})
+
+describe("SoldRecently", () => {
+  const trackEvent = jest.fn()
+
+  beforeEach(() => {
+    const mockTracking = useTracking as jest.Mock
+    mockTracking.mockImplementation(() => {
+      return {
+        trackEvent,
+      }
+    })
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const { getWrapper } = setupTestWrapper({
+    Component: (props: any) => {
+      return (
+        <MockBoot breakpoint={"lg" as Breakpoint}>
+          <tests.SoldRecentlyFragmentContainer
+            targetSupply={props.targetSupply}
```

Comment:
(non-blocking) Where does the `targetSupply` data actually get set? Is that paperwork taken care of in `setupTestWrapper`?

Reply:
Thats the beauty of our fancy new relay-test-tools (coolest new addition to force IMO):

https://github.com/artsy/force/blob/8b5b37f0c981b91035fec70f7b0129f6759e7946/src/v2/DevTools/setupTestWrapper.tsx#L39-L41

Mock payload generator will automatically generate fixture data. We can overwrite it by passing some mock resolvers in, but yeah, generally not needed.